callbacks/image_viewer.py

```diff
@@ -20,3 +25,68 @@ def render_image(img):
         width=620,
     )
     return fig
+
+
+@callback(
+    Output("image-src", "value"),
+    Output("image-selection-slider", "min"),
+    Output("image-selection-slider", "max"),
+    Output("image-selection-slider", "value"),
+    Output("image-selection-slider", "disabled"),
+    Input("image-src", "data"),
+)
+def update_slider_values(image_source_data):
```

Comment:
It might be worth making this logical while we wait for the data set up from `tiled`. Correct me if I'm wrong, but this callback will only run on page load right now? 

What if in the "Data selection" dropdown someone selects based on a list of directory names (rather than specific file names). Each directory could have the set of `.tiff` files that make up a given dataset. Then once someone makes that selection, this callback runs based on the contents of the directory that they've selected.

Reply:
The callback is trigger every time someone changes the dropdown with the img-source, yes.

However, it is ready for when that will include folders, not these file names. We simply add a dcc.Store which will be filled with the file names, when the data source is selected, and we will then use it here, to trigger this callback and use that data to plot the images.

Reply:
See c55295b commit in the PR, i have addressed the multi-project / folder + image selections